crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -115,15 +76,6 @@ pub struct Commands<'w, 's> {
     entities: &'w Entities,
 }
 
-impl SystemBuffer for CommandQueue {
-    #[inline]
-    fn apply(&mut self, _system_meta: &SystemMeta, world: &mut World) {
-        #[cfg(feature = "trace")]
-        let _span_guard = _system_meta.commands_span.enter();
-        self.apply(world);
-    }
-}
```

Comment:
Nit, but I think it makes sense to keep this impl here, since it's used only for the `Commands` definition just above